superwall/src/main/java/com/superwall/sdk/config/ConfigManager.kt

```diff
@@ -347,6 +348,59 @@ open class ConfigManager(
         }
     }
 
+    internal suspend fun refreshConfiguration() {
```

Comment:
This function needs to be a bit diff – don't update the status to retrying when we get config, need to reset paywall request cache (although this will likely change to be a bit more selective when Brian makes some changes). And remove the need to update config state on exception

Reply:
Updated to remove the status updates and added request cache resetting 👍